app/views/posts/index.html.erb

```diff
@@ -4,10 +4,10 @@
   <br>
   <br>
   <br>
-  <% @user.posts.each do |post| %>
+  <% @user.posts.includes(:comments).each do |post| %>
```

Comment:
- [ ] Great job adding the screenshots. As inicated by the previous reviewer, th best way of solving the N+1 problem is in the controllers and not in the views.  In the `posts#index` actions you should use `@posts = @user.posts.includes(:comments)`. You should not be fetching data from the views as rightly indicated by the previous reviewer. Kindly fix this issue.

Reply:
Oh, now I fully understand. Thanks @t-yanick